munimap/translations/messages.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
 "Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
-"POT-Creation-Date: 2021-06-29 16:33+0200\n"
+"POT-Creation-Date: 2021-07-01 13:59+0200\n"
```

Comment:
Just a question, but shouldn't a date not be hardcoded, or it is just a placeholder?

Reply:
The `messages.pot` and `messages.po` files are auto-generated and edited by hand. This just marks when it happened the last time.